src/Auth/BaseOAuthAuthentication.cs

```diff
@@ -1,139 +1,47 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.Linq;
 using System.Net.Http;
 using System.Net.Http.Headers;
-using System.Text;
 using System.Threading.Tasks;
 using IdentityModel.Client;
 
 namespace Jpp.Common.Backend.Auth
 {
     public abstract class BaseOAuthAuthentication : IAuthentication
     {
-        public bool Authenticated { get; private set; }
+        internal ErrorHandler _errorHandler;
```

Comment:
Don't think this is actually needed. A new instance could be created, as part of the HttpClient initialisation in GetAuthenticatedClient.

Reply:
Could be, but no reason to recreate if not needed

Reply:
Only a suggestion. I'd then just set it as private, and initialise from constructor.